test/sails.sockets.test.js

```diff
@@ -262,6 +262,11 @@ describe('low-level socket methods:', function (){
         });
       });
     });
+    it('should warn and not error if passing in null socket', function(done) {
+      // Testing fix for #6
+      sails.sockets.leave(null);
```

Comment:
What's the proper way to test that log.warn was called? 

Does your testing philosophy believe in testing for log calls in the first place?

I couldn't find Sinon or any spy library, so I just left it like this for now. Let me know if you want me to edit the test.